### PR TITLE
Update 'Apply' Section for Formatting and Terminology

### PR DESCRIPTION
**Pull request by NereidaRondon**
Resolves Issue #1

### CODE REVIEW

**Review by VidhiBhatt01 — COMMENTED** [review]
Raise a PR only after the issue is assigned,

PS: The issue is updated

**Comment by joeyouss** [comment]
@NereidaRondon I will get your PR reviewed and merged if it is correct.

**Comment by VidhiBhatt01** [comment]
@NereidaRondon So sorry. There was some internal confusion between we maintainers.

Anyways, I would suggest you that please  comment here once https://github.com/Codess-Cafe/Codess-Website/issues/1 so that we can assign it to you. Also, go through the description once again and update this PR accordingly

**Comment by NereidaRondon** [comment]
I went ahead and changed the extra requirement that was added to the issue. Hope all is good now. Thank you.

File: index.html [hunk 1]
```diff
@@ -912,8 +911,8 @@ <h3>Second Step</h3>
                 <div class="timeline-content">
                   <h3>Third Step</h3>
                   <p>
-                    Shortlisted members are contacted via mail, linkedIn or
-                    whatsapp within 14 days
+                    Shortlisted members are contacted via mail, Linkedin or
```
Comment (VidhiBhatt01) [inline index.html:914]: Linkedin -> LinkedIn

**Review by VidhiBhatt01 — CHANGES_REQUESTED** [review]
Just update one typo. Rest lgtm

**Comment by VidhiBhatt01** [comment]
Also please comment on #1

**Review by VidhiBhatt01 — APPROVED** [review]

**Pull request merged**